feature-libs/asm/customer-360/components/sections/components.ts

```diff
@@ -0,0 +1,14 @@
+/*
+ * SPDX-FileCopyrightText: 2023 SAP Spartacus team <spartacus-team@sap.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+export { AsmCustomerActiveCartComponent } from './asm-customer-active-cart/asm-customer-active-cart.component';
+export { AsmCustomerActivityComponent } from './asm-customer-activity/asm-customer-activity.component';
+export { AsmCustomerMapComponent } from './asm-customer-map/asm-customer-map.component';
+export { AsmCustomerProductInterestsComponent } from './asm-customer-product-interests/asm-customer-product-interests.component';
+export { AsmCustomerProductReviewsComponent } from './asm-customer-product-reviews/asm-customer-product-reviews.component';
+export { AsmCustomerProfileComponent } from './asm-customer-profile/asm-customer-profile.component';
+export { AsmCustomerSavedCartComponent } from './asm-customer-saved-cart/asm-customer-saved-cart.component';
+export { AsmCustomerSupportTicketsComponent } from './asm-customer-support-tickets/asm-customer-support-tickets.component';
```

Comment:
[MEDIUM REMARK] can be fixed in a patch release
1. all those components aren't in our public API now, because this file is not imported in `public_api.ts` (directly or transitively). So let's import this file in `public_api.ts`. Otherwise customers will have hard time importing those classes and customizing them.
2. Let's stick to the convetion of naming such "barrel" files to `index.ts` (which simply re-export API items higher in the folders structure). Let's change name of this file from `components.ts` to `index.ts`

ADDITIONAL ACTION NEEDED:
Please review the code in this PR and check if there other missing re-exports to the public API of our components/services etc.